feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -302,6 +302,7 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
     { customerId }: { customerId?: string },
     parameters?: AsmDeepLinkParameters
   ): void {
+    this.showCreateCustomerSuccessfullyAlert = false;
```

Comment:
maybe move to line 309 is better choice as other set/reset flag are in there(just avoid corner case the selected user  has no customer id. like  women@hybris.com)?